teamB/eungSooKim/app.js

```diff
@@ -53,11 +49,118 @@ app.post("/users", async (req, res, next) => {
             password
         ) VALUES (?, ?, ?);
         `,
-    [name, email, password]
+    [name, email, passWord]
   );
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts/write/:userId", async (req, res) => {
+  const { title, content } = req.body;
+  const userId = req.params.userId;
+
+  await database.query(
+    `INSERT INTO posts(
+            title,
+            content,
+            user_id
+        ) VALUES (?, ?, ${userId});
+        `,
+    [title, content]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/posts/all", async (req, res) => {
```

Comment:
```suggestion
app.get("/posts", async (req, res) => {
```
http method 의 get은 무언가의 데이터를 받아오는 요청이고 post**s**(게시글 `'들'`)이라는 복수의 의미이기 때문에 위와 같이 작성되어도 충분히 의미를 전달할 수 있는 endpoint가 될 것 같습니다-!